src/ck-cp/controlPoints.C

```diff
@@ -693,7 +693,7 @@ void controlPointManager::setFrameworkAdvancePhase(bool _frameworkShouldAdvanceP
 #if CMK_LBDB_ON && 0
     // TODO: This code is ifdef'd out but is broken, the list of LB objects will
     // likely have nullptrs in it, which this doesn't check for
-    LBDatabase * myLBdatabase = LBDatabaseObj();
+    LBManager * myLBdatabase = LBManagerObj();
```

Comment:
Why is it named LBdatabase? Also, semantically, should this have been changed? Seems like getting a list of objects would be something you'd ask the LBDatabase for? Possibly somewhere in this change (if I didn't miss it) there should be a high level comment explaining the exact responsibilities of the LBDatabase and the LBManager, and the distinction between them.